android/app/src/main/java/org/cryptimeleon/incentive/app/NavGraph.kt

```diff
@@ -15,6 +15,7 @@ import androidx.navigation.NavType
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.navArgument
+import org.cryptimeleon.incentive.app.ui.attacker.AttackerUi
```

Comment:
AttackerUI is the UI you show on the PR page, I assume

Reply:
Indeed it is